.github/workflows/release-backwards-compatiblity.yml

```diff
@@ -21,10 +22,33 @@ env:
   MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
 
 jobs:
+  check-backward-compatibility-run:
+    name: Check if backward compatibility check is needed for the version we want to release
+    runs-on: ubuntu-latest
+    outputs:
+      is-beta: ${{ steps.check-if-beta-version.outputs.is-beta }}
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check if beta version
+        id: check-if-beta-version
+        run: |
+          current_version=$(.github/script/get-current-version)
+          if [[ $current_version == *"beta"* ]]; then
+            echo "::set-output name=is-beta::yes"
+            echo "Is beta"
+          else
+            echo "::set-output name=is-beta::no"
+          fi
+          
+
   release-integ-test:
     name: Run Backwards Compatible Integration Tests with Release Tarball
     runs-on: ubuntu-latest
-
+    needs: check-backward-compatibility-run
+    if: ${{ needs.check-backward-compatibility-run.outputs.is-beta == 'no' }}
```

Comment:
Can we create  a script to get prev_version (in node) and if that script returns no then quit.

Reply:
Yes, saw the slack message was just testing. will do :)

Reply:
Created a node script and just call that in GA.